README.md

```diff
@@ -88,7 +101,405 @@ workflow wf {
 }
 ```
 
-Architecture
-------------
+# Getting Started with WDL
+
+We'l use Cromwell to run these examples but you can use any WDL engine of your choice. If you don't already have a reference to the Cromwell JAR file, one can be [downloaded](https://github.com/broadinstitute/cromwell/releases) 
+
+## Hello World WDL
+
+Create a WDL simple file and save it as `hello.wdl`, for example:
+
+```
+task hello {
+  String name
+
+  command {
+    echo 'hello ${name}!'
+  }
+  output {
+    File response = stdout()
+  }
+}
+
+workflow test {
+  call hello
+}
+```
+
+Generate a template `hello.json` file with the `inputs` subcommand:
+
+```
+$ java -jar cromwell.jar inputs hello.wdl
+{
+  "test.hello.name": "String"
+}
+```
+
+WDL has a concept of fully-qualified names.  In the above output, `test.hello.name` is a fully-qualified name which shoul\
```

Comment:
The slashes at the EOL are visible in the [README](https://github.com/broadinstitute/wdl/tree/1e14be65402b2b43e7b6d1bff0ae7100521f85a1#hello-world-wdl)